assets/js/app.js

```diff
@@ -74,40 +116,27 @@ app.controller('controller', function($scope, $rootScope, $notification, $storag
       io.socket._raw.emit('username', newUsername);
     });
 
-    $scope.getAllVideos = function() {
-      io.socket.get('/video/subscribe');
-
-      $http.get('/api/start').success(function(config) {
-        $scope.videos = config.videos;
-        $scope.autoplay = config.autoplay;
-      });
-    };
-
-    $scope.getAllVideos();
+    $video.getAll()
+      .then((config) => $scope.autoplay = config.autoplay);
+    $video.subscribe();
 
     io.socket.on('video', function(obj) {
```

Comment:
i'll eventually want to move this socket listener into the video service as well but let's just relax a bit